frontend/src/utils/api.tsx

```diff
@@ -97,6 +97,66 @@ export const getPipelines = async (filter: string, skip: number, limit: number,
     }
 }
 
+
+export const checkPipelineValidity = async (jsonDescription: string)=> {
+    /*
+     * Function used to check if a pipeline is valid (using a Json representation of said pipeline)
+     * jsonDescription: string - the pipeline's description, in json format.
+     */
+
+    const headers = {
```

Comment:
Are these headers mandatory ? or could we use the const variable on top of the file ?